### PR TITLE
(BE #1) Add relaxed dependency and converter router

### LINKED ISSUE #1: Add relaxed dependency and converter router

### PR DESCRIPTION
**Pull request by elroykanye**

### CODE REVIEW

**Review by Burnleydev1 — CHANGES_REQUESTED** [review]
I have left some few comments, but I think everything else is fine.

File: public/javascripts/html2pdf.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                this.relaxedGlobals,
+                defaultTempHtmlPath,
+                defaultOutputPdfPath,
+                json_data
+            );
+        }
+    }
+}
+
+module.exports = new HTML2PDF();
```
Comment (Burnleydev1) [inline public/javascripts/html2pdf.js]: please ensure that there is an empty line at the end

**Review by nebadesmondc — APPROVED** [review]
LG!

**Review by Burnleydev1 — APPROVED** [review]
LGTM

**Pull request merged**